frontend/app/components/page-header-brand.tsx

```diff
@@ -3,18 +3,33 @@ import { useTranslation } from 'react-i18next';
 import { LanguageSwitcher } from '~/components/language-switcher';
 import { getAltLanguage } from '~/utils/locale-utils';
 
-export function PageHeaderBrand() {
+export interface PageHeaderBrandProps {
+  link?: boolean;
```

Comment:
I don't think 'link' is the best choice here. I would recommend using a more explicit name for the prop and making it required.

My suggestion is to use a prop called headerLogoVariant with two options.

```suggestion
  /**
   * Determines how the logo is displayed in the header.
   * 
   * @type {'logo-only' | 'logo-with-link'}
   * @description
   *   - `'logo-only'`: Displays the logo without a link.
   *   - `'logo-with-link'`: Displays the logo wrapped in a link.
   */
  headerLogoVariant: 'logo-only' | 'logo-with-link';
```

OR

`headerLogoUrl` prop: If this prop is defined and non-empty,  the logo will be wrapped in a link pointing 
to this URL.

```suggestion
  /**
   * The URL to wrap the logo with. If this prop is defined and non-empty, 
   * the logo will be wrapped in a link pointing to this URL.
   * 
   * @type {string}
   */
  headerLogoUrl?: string;
```

Reply:
imo the `headerLogo` prefix is kinda redundant.. just use `variant`
alternatively, why not move the link to the parent component? `<Link><PageHeaderBrand/></Link>`

Reply:
I don't think we want to wrap the whole `<PageHeaderBrand>` We just want to wrap the logo itself. So I'll take the `variant` prop approach.